app/src/main/java/ca/marshallasch/veil/database/KnownPostsContract.java

```diff
@@ -12,6 +12,11 @@
  */
 public final class KnownPostsContract {
 
+    //Post status ints
+    public static int POST_NORMAL = 0;
```

Comment:
These fields should be final so they can not be set elsewhere in the app.

Reply:
resolved